frontend/public/images/whatNewSection_bg.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Remove this SVG in web is not require this info.
If I remember correct Adobe Illustrator have option to export SVG for web that remove unnecessary info